docs/topics/about-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id='about-intellij-idea-plugin_{context}']
+= About the {ProductShortName} plugin for IntelliJ IDEA
+
+The {ProductName} ({ProductShortName}) plugin for IntelliJ IDEA helps you migrate and modernize applications.
```

Comment:
See my comments below about the "X product does Y" wording. We need to focus on the user.

Reply:
I think this would be a good place to mention both editions of the IDEA.